src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
```

Comment:
- [ ] Please remove this import index.css code since it cause errors when the user try to init the app